### PR TITLE
Feaure/iii 3666 copy changes saved searches

### PR DESCRIPTION
**Pull request by simon-debruijn**
### Added
- fr translations
---
Ticket: https://jira.uitdatabank.be/browse/III-3666

### CODE REVIEW

File: src/saved-searches/ui/saved-searches-list.html [hunk 1]
```diff
@@ -1,9 +1,9 @@
 <div class="container-fluid">
-    <h1>Mijn zoekopdrachten</h1>
+    <h1 translate-once="search.savedSearches.mySearches"></h1>
     <table class="table">
         <tr>
             <th class="saved-search-title-column">
-                <strong>Titel</strong>
+                <strong translate-once="search.savedSearches.title"></strong>
```
Comment (brampauwelyn) [inline src/saved-searches/ui/saved-searches-list.html:6]: Shouldn't this be `search.savedSearches.savedSearches` ?

File: src/core/translations/french-translations.constant.js [hunk 2]
```diff
@@ -1096,10 +1096,13 @@ angular.module('udb.core')
       search: 'Chercher',
       advanced: 'Avancé',
       savedSearches: {
-        title: 'Recherches conservées',
```
Comment (brampauwelyn) [inline src/core/translations/french-translations.constant.js:1099]: I prefer the `title` approach

File: src/search/components/search-bar.directive.html [hunk 3]
```diff
@@ -6,7 +6,7 @@ <h1 class="title" translate-once="search.search">Zoeken</h1>
     <span class="dropdown saved-search-icon" uib-dropdown>
       <i class="fa fa-bookmark" class="dropdown-toggle" uib-dropdown-toggle></i>
       <ul class="dropdown-menu" uib-dropdown-menu role="menu">
-        <li role="presentation" class="dropdown-header" translate-once="search.savedSearches.title">Bewaarde zoekopdrachten</li>
+        <li role="presentation" class="dropdown-header" translate-once="search.savedSearches.savedSearches">Bewaarde zoekopdrachten</li>
```
Comment (brampauwelyn) [inline src/search/components/search-bar.directive.html:9]: Remove the text between the tags

**Review by brampauwelyn — DISMISSED** [review]

File: src/core/translations/french-translations.constant.js (hunk 2 above)
Comment (simon-debruijn) [reply src/core/translations/french-translations.constant.js:1099]: see other key 'title'

File: src/saved-searches/ui/saved-searches-list.html (hunk 1 above)
Comment (simon-debruijn) [reply src/saved-searches/ui/saved-searches-list.html:6]: No it's literally 'Titel' and 'Titre' as a new key 'title', that's why I changed the other key

**Review by jdrieghe — APPROVED** [review]

**Review by bertramakers — APPROVED** [review]

**Comment by bertramakers** [comment]
I agree with @simon-debruijn, we can't use the `title` key for `search.savedSearches.savedSearches` since there's also a `Title` text

**Pull request merged**